src/navigation/wallet/screens/GlobalSelect.tsx

```diff
@@ -263,9 +264,8 @@ const GlobalSelect: React.FC<GlobalSelectProps> = ({
     if (recipient?.currency) {
       wallets = wallets.filter(
         wallet =>
-          wallet.currencyAbbreviation === recipient?.currency ||
-          (recipient?.opts?.showERC20Tokens &&
-            IsERCToken(wallet.currencyAbbreviation, wallet.chain)),
+          wallet.currencyAbbreviation === recipient?.currency &&
+          wallet.chain === recipient?.chain,
```

Comment:
this condition could be breaking cases in which the chain does not exist

Reply:
Thanks! Just added another condition to avoid crash if `chain` no exists.